File: c/1005.c

// https://www.urionlinejudge.com.br/judge/pt/problems/view/1005

#include <stdio.h>
#include <stdlib.h>

int main()
{
    double a, b, media;

    scanf("%lf %lf", &a, &b);

    if ((a >= 0.0) && (a <= 10.0) && (b >= 0.0) && (b <= 10.0)){
        media = ((a * 3.5) + (b * 7.5)) / 11;

        printf("MEDIA = %.5lf\n", media);
    }

    return 0;
}

File: haskell/1005.hs

-- https://www.urionlinejudge.com.br/judge/pt/problems/view/1005

import Text.Printf (printf)

main :: IO ()
main = do a <- readLn :: IO Double
          b <- readLn :: IO Double
          printf "MEDIA = %.5f\n" ((3.5*a + 7.5*b)/11)